pandas/tests/io/excel/test_readers.py

```diff
@@ -420,8 +420,10 @@ def test_reader_special_dtypes(self, request, read_ext):
                     datetime(2013, 12, 14),
                     datetime(2015, 3, 14),
                 ],
+                "BoolColWithBlank": [True, False, None, True, False],
```

Comment:
@datapythonista I upated the test_types files as you suggested but for some reason no False or True values are shown. Only 1 or 0 and I cannot find out why.